src/data/events/highlights.js

```diff
@@ -25,7 +25,7 @@ const highlightedEvents = [
 	{
 		name: 'Hack on the Hill X',
 		description: HOTHXDescription,
-		imgFilePath: 'event/2023w-hoth-x-banner.png', // TODO: update banner
+		imgFilePath: 'event/2023w-hoth-x-banner.png',
```

Comment:
You should be able to create another button here to say “sign up” (I think it’s `rsvp` but I could be wrong) that allows you to link the rsvp form

Reply:
it seems like the highlights don't have the capability to have 2 buttons. the rsvp link is in the event card, however. let me know if I'm wrong LOL

Reply:
Huh. I must be tripping then. In that case can you change the text on the button from 'More Info' to 'Sign Up'? Ty :)